### ISSUE TITLE
PermissionDenied trying to call chat() or any other service method

### ISSUE THREAD

**Issue by hm-ca** [google/generative-ai-python#10]
Getting the following error trying to call execute this code:
```
import google.generativeai as genai
import os

genai.configure(api_key=os.environ['GOOGLE_API_KEY'])

response = genai.chat(messages=["Hello."])
```

**Exception has occurred: PermissionDenied**       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
403 Generative Language API has not been used in project 411082643095 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/generativelanguage.googleapis.com/overview?project=411082643095 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry. [links {
  description: "Google developers console API activation"
  url: "https://console.developers.google.com/apis/api/generativelanguage.googleapis.com/overview?project=411082643095"
}
, reason: "**SERVICE_DISABLED**"
domain: "googleapis.com"
metadata {
  key: "consumer"
  value: "projects/411082643095"
}
metadata {
  key: "service"
  value: "generativelanguage.googleapis.com"
}
]

**Comment by MarkDaoust** [comment]
Hi, Thanks for reaching out.

That error is trying to tell you that in your project you still needs to enable the api.

- Have you been through the [waitlist](https://makersuite.google.com/waitlist)?
- Is the "Generative Language API" enabled on your project's "Enabled APIs & Services" page?

**Comment by thirdwing** [comment]
I got the same error when calling the embedding API. I have Vertex AI enabled.

**Comment by MarkDaoust** [comment]
Yeah, sorry, this is a separate API, unrelated to vertex, you'll need to sign up on the waitlist, then enable the api on your project. 

The wait-list shouldn't take long.
